app/src/test/java/org/oppia/android/app/testing/HomeSpanTest.kt

```diff
@@ -167,6 +168,7 @@ class HomeSpanTest {
   }
 
   private fun createHomeActivityIntent(profileId: Int): Intent {
+    val profileId = ProfileId.newBuilder().setInternalId(profileId).build()
```

Comment:
Please differentiate between profileId and internalprofileId to avoid ambiguity.

Reply:
Okay.